pkg/operator/ceph/csi/spec.go

```diff
@@ -683,3 +739,116 @@ func (r *ReconcileCSI) validateCSIVersion(ownerInfo *k8sutil.OwnerInfo) (*CephCS
 	}
 	return version, nil
 }
+
+func (r *ReconcileCSI) configureHolder(enabledDrivers []driverDetails, tp templateParam, pluginTolerations []corev1.Toleration, pluginNodeAffinity *corev1.NodeAffinity) error {
+	for _, multusCluster := range r.multusClusters {
+		for _, driver := range enabledDrivers {
+			cephPluginHolder, err := templateToDaemonSet("cephpluginholder", driver.holderTemplate, tp)
```

Comment:
nit: What about a helper method for this block? There something about a long block of code inside a double for loop that seems like it would help readability.